ShelterModule/Models/Announcements/AnnouncementResponse.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace ShelterModule.Models.Announcements;
+
+public sealed class AnnouncementResponse
```

Comment:
We should mark required (non-nullable) fields with `[Required]` attribute as well, because Swagger does not access `required` keywords on properties, only attributes.